client/src/dojo/modelManager/utils/LeaderboardUtils.ts

```diff
@@ -1,22 +1,7 @@
 import { ClientComponents } from "@/dojo/createClientComponents";
-import {
-  EternumGlobalConfig,
-  HYPERSTRUCTURE_POINTS_ON_COMPLETION,
-  HYPERSTRUCTURE_RESOURCE_MULTIPLIERS,
-  HYPERSTRUCTURE_TOTAL_COSTS_SCALED,
-  ResourcesIds,
-} from "@bibliothecadao/eternum";
+import { EternumGlobalConfig, HYPERSTRUCTURE_RESOURCE_MULTIPLIERS, ResourcesIds } from "@bibliothecadao/eternum";
 import { ComponentValue } from "@dojoengine/recs";
-
-export const TOTAL_CONTRIBUTABLE_AMOUNT: number = HYPERSTRUCTURE_TOTAL_COSTS_SCALED.reduce(
-  (total, { resource, amount }) => {
-    return (
-      total +
-      (HYPERSTRUCTURE_RESOURCE_MULTIPLIERS[resource as keyof typeof HYPERSTRUCTURE_RESOURCE_MULTIPLIERS] ?? 0) * amount
-    );
-  },
-  0,
-);
+import { ClientConfigManager } from "../ConfigManager";
 
 function getResourceMultiplier(resourceType: ResourcesIds): number {
   return HYPERSTRUCTURE_RESOURCE_MULTIPLIERS[resourceType] ?? 0;
```

Comment:
this is in config object - let's replace it too

Reply:
This is a workaround for client tests, otherwise they break atm